packages/react-magma-dom/package.json

```diff
@@ -40,25 +40,26 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-magma-icons": "^3.0.0",
-    "uuid": "^8.3.0"
+    "uuid": "^9.0.1"
   },
   "dependencies": {
-    "@emotion/react": "^11.13.0",
+    "@emotion/react": "^11.13.3",
     "@emotion/styled": "^11.13.0",
     "@floating-ui/react-dom": "^2.1.2",
-    "@popperjs/core": "^2.6.0",
-    "csstype": "^3.0.8",
-    "polished": "^3.2.0",
-    "react-popper": "^2.2.4"
+    "@popperjs/core": "^2.11.8",
+    "@types/uuid": "^9.0.1",
+    "csstype": "^3.1.3",
+    "polished": "^3.7.2",
+    "react-popper": "^2.3.0"
   },
   "devDependencies": {
-    "date-fns": "^2.16.1",
-    "downshift": "^5.4.5",
-    "framer-motion": "^4.1.11",
+    "date-fns": "^2.30.0",
+    "downshift": "^5.4.7",
+    "framer-motion": "^4.1.17",
     "mkdirp": "^1.0.4",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-magma-icons": "^3.0.0",
-    "uuid": "^8.3.0"
+    "uuid": "^9.0.1"
```

Comment:
If we are adding this change, it needs to be called out specifically in the changeset because it can affect our adopters